middleware/admin-auth.js

```diff
@@ -0,0 +1,14 @@
+const adminAuth = (req, res, next) => {
+  if (req.path === '/create-user') {
```

Comment:
please write a proper middleware to apply to admin routes. why do you make this a global middleware. please don't just copy someone else's code and paste. this is a wrong way of doing this

Reply:
lol sir. i didnt copy code o, if i did, i would have done it correctly(winks). but i dont really understand what you want me to do here sha. i have refactored the codes, i hope i do it well this time